### PR TITLE
Add additional documentation steps for setup

### PR DESCRIPTION
**Pull request by nring**
These additional steps were the blockers for me during set up. Please double check that I have any nomenclature listed correctly. This is very new to me!

### CODE REVIEW

File: services/api/README.md [hunk 1]
```diff
@@ -20,16 +20,33 @@ Install all dependencies from the root of the monorepo:
...
 ```
 
 Per the `.env`, you'll also need to be connected to an Algorand node, whether in development or production.
 
+### Creating a local Algorand Sandbox account
 For local development, the [Algorand Sandbox](https://github.com/algorand/sandbox) is handy docker instance that makes interfacing with the blockchain simple from your local machine. It will also create an account that can be funded with fake Algos using the [Testnet Dispenser](https://dispenser.testnet.aws.algodev.network/).
- 
+
+To create an account:
+* Download the [Alogrand Sandbox] and start up the docker instance:
```
Comment (smonn) [inline services/api/README.md:35]: Just making it a valid link (if that was the intention):

```suggestion
* Download the [Alogrand Sandbox](https://github.com/algorand/sandbox) and start up the docker instance:
```

Alternatively, remove the brackets:

```suggestion
* Download the Alogrand Sandbox and start up the docker instance:
```

**Review by smonn — APPROVED** [review]
All good. The DB name doesn't matter as long as it matches what's in the `.env` files. One other gotcha to look out for is the DB schema name. If not using the default `public`, you'll need to run `CREATE SCHEMA name` to create it before applying migrations.

File: apps/cms/README.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ The purpose of the CMS is to provide content authors the ability to create and m
 
 ## Initial setup
 
-Once the environment variables have been added to the `.env` file, you'll want to create an admin user, which will allow you to login and administer the CMS:
+Once the environment variables have been added to the `.env` file, you'll want to create a new Postgres database named `algo-cms`.
```
Comment (jakerainis) [inline apps/cms/README.md:9]:
```suggestion
Once the environment variables have been added to the `.env` file, you'll want to create a new Postgres database with the name specified in the `.env` file.
```

**Review by jakerainis — APPROVED** [review]
Thanks for doing this!

File: apps/cms/README.md [hunk 3]
```diff
@@ -40,23 +42,25 @@ After bootstrapping and running the seed script locally, there's additional conf
...
 ### 2. Add token for user
 
 In order for the API to connect to the CMS, the Directus key needs to be inputted into the CMS:
 
-a. Go to your profile (the icon at the very bottom left)
-b. Scroll all the way down to Token, under Admin Options
-c. Paste in the desired API key (has to match `CMS_API_KEY` in web)
+1. Go to your profile (the icon at the very bottom left)
+1. Scroll all the way down to Token, under Admin Options
+1. Paste in the desired API key (has to match `CMS_ACCESS_TOKEN` in web)
```
Comment (afraser) [inline apps/cms/README.md:63]: I don't see this env var. I do see `CMS_API_KEY`

Comment (afraser) [reply apps/cms/README.md:63]: oh `CMS_ACCESS_TOKEN` is in `services/api/.env`

File: services/api/README.md [hunk 4]
```diff
@@ -20,16 +20,41 @@ Install all dependencies from the root of the monorepo:
...
- 
+
+To create an account:
+* Download the [Algorand Sandbox](https://github.com/algorand/sandbox) and start up the docker instance:
+```bash
+./sandbox up
+```
+* Then create an account:
+```bash
+./sandbox goal account new
```
Comment (afraser) [inline services/api/README.md:49]: Yes! Thank you!

File: apps/cms/README.md (hunk 3 above)
Comment (afraser) [reply apps/cms/README.md:63]:
```suggestion
1. Paste in the desired API key (has to match `CMS_ACCESS_TOKEN` in services/api/.env)
```

**Pull request merged**